src/tools/clippy/clippy_utils/src/sugg.rs

```diff
@@ -265,8 +266,9 @@ impl<'a> Sugg<'a> {
                 snippet_with_context(cx, lhs.span, ctxt, default, app).0,
                 snippet_with_context(cx, ty.span, ctxt, default, app).0,
             ),
+            //FIXME(chenyukang), remove this after type ascription is removed from AST
             ast::ExprKind::Type(ref lhs, ref ty) => Sugg::BinOp(
-                AssocOp::Colon,
+                AssocOp::As,
```

Comment:
Any better way for this? 🤔

Reply:
I'd do it the same as others above, just get a snippet